BO4E/COM/Angebotsteil.cs

```diff
@@ -50,5 +50,11 @@ public class Angebotsteil : COM
         [JsonPropertyName("positionen")]
         [ProtoMember(7)]
         public List<Angebotsposition> Positionen { get; set; }
+
+        /// <summary>Hier kann der Belieferungszeitraum angegeben werden, für den dieser Angebotsteil gilt. Details <see cref="Zeitraum" /></summary>
+        [JsonProperty(PropertyName = "lieferzeitraum", Required = Required.Default)]
+        [JsonPropertyName("lieferzeitraum")]
+        [ProtoMember(8)]
+        public Zeitraum Lieferzeitraum { get; set; }
```

Comment:
Benutzen wir nullable typen? Default ist doch nullable oder? 
```suggestion
        public Zeitraum? Lieferzeitraum { get; set; }
```

Reply:
Bei den Zeiträumen scheinbar nicht.
![grafik](https://user-images.githubusercontent.com/73470827/155500831-364d63f7-5cf9-4790-824a-7c61471b3596.png)